### PR TITLE
exim: Initial Addition

### PR DESCRIPTION
**Pull request by kingthorin**
Initial add-on with migration of saveraw and savexml.

There is a fair amount of overlap between those two classes, I'll look at making an Abstract or something to reduce the duplication. (Next PR.)

Also note if anyone in the core goes ahead with testing this you should remove saverawmessage or savexmlmessage first, as nothing has been done to address the duplicate menu topic.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/exim/exim.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+version = "0.0.1"
+description = "Import and Export functionality"
+
+zapAddOn {
+    addOnName.set("Import/Export")
+    addOnStatus.set(AddOnStatus.RELEASE)
```
Comment (thc202) [inline addOns/exim/exim.gradle.kts:8]: `ALPHA`

File: addOns/exim/exim.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    zapVersion.set("2.10.0")
+
+    manifest {
+        author.set("ZAP Dev Team")
+        url.set("https://www.zaproxy.org/docs/desktop/addons/save-raw-message/")
```
Comment (thc202) [inline addOns/exim/exim.gradle.kts:13]: Needs tweak, should be `.../addons/import-export/`.

File: addOns/exim/src/main/javahelp/help/helpset.hs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE helpset
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp HelpSet Version 2.0//EN"
+         "http://java.sun.com/products/javahelp/helpset_2_0.dtd">
+<helpset version="2.0" xml:lang="en-GB">
+  <title>Save Raw Message</title>
```
Comment (thc202) [inline addOns/exim/src/main/javahelp/help/helpset.hs:6]: Needs tweak.

File: addOns/exim/src/main/javahelp/help/helpset.hs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  <maps>
+     <homeID>saverawmessage</homeID>
```
Comment (thc202) [inline addOns/exim/src/main/javahelp/help/helpset.hs:9]: `exim`

**Comment by kingthorin** [comment]
Thanks

**Comment by kingthorin** [comment]
Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**